lib/travis/build/script/r.rb

```diff
@@ -450,8 +450,8 @@ def setup_latex
             mactex = 'BasicTeX.pkg'
             # TODO(craigcitro): Confirm that this will route us to the
             # nearest mirror.
-            sh.cmd 'wget http://mirror.ctan.org/systems/mac/mactex/'\
-                   "#{mactex} -O \"/tmp/#{mactex}\""
+            sh.cmd 'curl -fLo \"/tmp/#{mactex}\" --retry 3 http://mirror.ctan.org/systems/mac/mactex/'\
+                   '#{mactex}'
```

Comment:
Pretty sure the single quotes here should be double quotes, single quotes are not expanded by the shell.